### ISSUE TITLE
[Bug] tune different parameters but get the same output accuracy

### ISSUE THREAD

**Issue by Waterkin** [ray-project/ray#23493]
### Search before asking

- [X] I searched the [issues](https://github.com/ray-project/ray/issues) and found no similar issues.

### Ray Component

Ray Tune

### Issue Severity

High: It blocks me to complete my task.

### What happened + What you expected to happen

As the title says, I tune different parameters for the same LSTM model(hidden_size, batch_size, learning rate...), but finally I got the same output, it is unbelievable!

### Versions / Dependencies

Newest Ray tune
Linux Ubuntu 18.0.1
Python 3.8
Pytorch 1.8
Cuda 11.4

### Reproduction script

As the title says, I tune different parameters for the same LSTM model(hidden_size, batch_size, learning rate...), but finally I got the same output, it is unbelievable!

### Anything else

As the title says, I tune different parameters for the same LSTM model(hidden_size, batch_size, learning rate...), but finally I got the same output, it is unbelievable!

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

**Comment by krfricke** [comment]
Hi @Waterkin, can you provide more context here, please - how does your training script look like, how do your results look like? We can't help you if you don't provide us enough information.

**Comment by krfricke** [comment]
Closing this as it doesn't have a repro script - feel free to open once you do!
